applications/StructuralMechanicsApplication/custom_elements/cr_beam_element_linear_3D2N.cpp

```diff
@@ -51,6 +51,44 @@ CrBeamElementLinear3D2N::Create(IndexType NewId,
 
 CrBeamElementLinear3D2N::~CrBeamElementLinear3D2N() {}
 
+void CrBeamElementLinear3D2N::Initialize(const ProcessInfo& rCurrentProcessInfo)
+{
+    KRATOS_TRY
+
+    if (this->GetProperties().Has(SEMI_RIGID_NODE_IDS)) {

```

Comment:
in a restart, should this be done every time?

I guess not no?

Reply:
this cannot be done anymore, as the check for semi-rigidity cannot be done in Initialize anymore, since there is the possibility that the rotational stiffness is not assigned yet (depending on the order of the proces executions